server/src/main/java/org/candlepin/model/ImportUpstreamConsumer.java

```diff
@@ -84,6 +84,10 @@
     @Size(max = 255)
     private String prefixUrlApi;
 
+    @Column(length = 255, name = "content_access_mode")
```

Comment:
`length` here doesn't really do anything.  That property is only used, IIRC, for DDL generation which we don't do since we have Liquibase.